UI/DropsController.gd

```diff
@@ -9,11 +9,13 @@ const NO_DROPS_OPTION_ID = 4096
 export (PackedScene) var drop_item_panel_packed_scene: PackedScene = preload("res://UI/DropItemPanel.tscn")
 
 var _selected_drops_table: DropsTable = null
+var _current_filter_text: String = ("")
```

Comment:
Instead of storing it locally you could do $DropsFilterLineEdit.text, but its fine either way